agents/agents-features/agents-features-memory/src/jvmMain/kotlin/ai/koog/agents/memory/storage/Aes256GCMStorageEncryptor.kt

```diff
@@ -1,7 +1,7 @@
 package ai.koog.agents.memory.storage
 
 import java.security.SecureRandom
-import java.util.*
+import java.util.Base64
```

Comment:
A good practice would be to use the Kotlin api where possible. Although in this case it doesn’t change anything, since kotlin simply wraps Java `Base64`. This approach will reduce potential issues if the code ever needs to be moved to common